src/stores/canvas.ts

```diff
@@ -41,6 +45,16 @@ const useStoreCanvas = defineStore({
       })
       await promise
     },
+    async resetCanvas(stageParentDiv: HTMLDivElement) {
+      // delete
+      useStoreLine().deleteLines()
+      useStoreText().deleteTexts()
+      useStoreImage().deleteImages()
+      // reset history
```

Comment:
```suggestion
      // save history
```
リセットではないのでコメント消すか変更をお願いします。

Reply:
コメント変更しました

Reply:
コメント変更しました